database/migrations/2024_11_18_130759_produtos.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('produtos', function (Blueprint $table) {
+            $table->id();
+            $table->bigInteger('categoria_id');
+            $table->bigInteger('fornecedor_id');
+            $table->string('sku');
+            $table->string('nome')->index();
+            $table->float('valor');
+            $table->float('valor_promocional')->nullable();
+            $table->bigInteger('quantidade');
```

Comment:
Pode ser melhor colocar como float, se for uma quantidade em metros?

Reply:
Coloquei quantidade como float e coloquei as casas decimais. Exemplo: 3,75 metros de corda